### DISCUSSION TITLE
How to reinstall the libzip distribution when build lumen in docker

### DISCUSSION THREAD

**Discussion by luluhoward** [codingreshapefuture/Community#30] [Q&A] [answered]
I use docker version Version 17.12.0-ce-mac55 (23011) on osx version 10.11.6. I have a problem when I build lumen on docker, but when the build process is finished there is an error like this :

```configure: error: Please reinstall the libzip distribution ERROR: Service 'app' failed to build: The command '/bin/sh -c apt-get update && apt-get install -y libpng-dev libjpeg-dev libpq-dev && rm -rf /var/lib/apt/lists/* && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr && docker-php-ext-install gd mbstring pdo pdo_mysql pdo_pgsql zip' returned a non-zero code: 1```

This is my dockerfile settings:

![image](https://user-images.githubusercontent.com/103616225/192132587-8547076d-43b1-424c-8909-9c038f3e4982.png)

So the build process fails. Has anyone ever had a case like me? I hope someone can provide a solution.

**Comment by lucthienphong1120** [comment, marked as answer]
Try to configure zip with libzip and install libzip-dev

```
#install some base extensions
RUN apt-get install -y zip libzip-dev && docker-php-ext-configure zip --with-libzip && docker-php-ext-install zip
```

**Reply by luluhoward** [reply]
Using FROM php:7.3-apache this is what finally worked for me. Thanks!
